src/guiapi/src/MItem_menus.cpp

```diff
@@ -218,3 +220,38 @@ MI_FOOTER_SETTINGS::MI_FOOTER_SETTINGS()
 void MI_FOOTER_SETTINGS::click(IWindowMenu & /*window_menu*/) {
     Screens::Access()->Open(GetScreenMenuFooterSettings);
 }
+
+MI_LANGUAGUE_USB::MI_LANGUAGUE_USB()
+    : WI_LABEL_t(_(label), 0, is_enabled_t::yes, is_hidden_t::no) {}
+
+void MI_LANGUAGUE_USB::click(IWindowMenu &windowMenu) {
+    if (fileProviderUSB.OpenFile())
+        ProviderRegistrator("ts", &fileProviderUSB);
```

Comment:
I'd recommend calling the registration via a function here - declaring a temporary and unnamed object may end up being optimized away in release builds.
`Translations::Instance().RegisterProvider(Translations::MakeLangCode("ts"), &fileProviderUSB);`